### ISSUE TITLE
Error when trying to use binding

### ISSUE THREAD

**Issue by pianocomposer321** [hkupty/nvimux#45]
Steps to reproduce:
Press prefix-" or prefix-%. May happen with other bindings too, I haven't checked.

Error:
```
E5108: Error executing lua ...e/nvim/site/pack/packer/start/nvimux/lua/nvimux/vars.lua:18: Expected lua t
able                                                                                                     
stack traceback:                                                                                         
        [C]: in function 'nvim_buf_set_lines'                                                            
        ...e/nvim/site/pack/packer/start/nvimux/lua/nvimux/vars.lua:18: in function 'singleton_buf'      
        ...e/nvim/site/pack/packer/start/nvimux/lua/nvimux/vars.lua:58: in function 'select_buffer'      
        ...e/nvim/site/pack/packer/start/nvimux/lua/nvimux/init.lua:83: in function <...e/nvim/site/pack/
packer/start/nvimux/lua/nvimux/init.lua:71>                                                              
        ...im/site/pack/packer/start/nvimux/lua/nvimux/bindings.lua:75: in function 'do_mapping'         
        [string ":lua"]:1: in main chunk
```

**Comment by robobenklein** [comment]
Getting a similar thing with neovim version `v0.7.0-dev+895-ga0201b6ed`, v0.4.4 (ubuntu apt version) could not setup the plugin.

`E5108: Error executing lua: /home/robo/.vim/plugged/nvimux/lua/nvimux/vars.lua:18: Expected lua table`

**Comment by pianocomposer321** [comment]
Looking at the source code, it would appear that the error is happening because the function `vim.api.nvim_buf_set_lines` takes a table as its last argument, but something else is supplied. On line 11, you check to see if a variable equals "table", and if it does, you set content to equal that variable, thereby setting content to equal "table", which is a string. Without being able to test it right now, I suspect that this is the source of the problem. I think changing line 12 from `content = tp` to `content = vars.scratch_buf_content` may solve the problem. I will test this when I get a chance, assuming @hkupty doesn't solve it before then.

**Comment by hkupty** [comment]
Thanks for the issue. I'll check it out.

Are you using nightly/master or latest release?

**Comment by pianocomposer321** [comment]
It would appear that someone else has already seen this and opened a PR (#42).

**Comment by pianocomposer321** [comment]
> Thanks for the issue. I'll check it out.
> 
> Are you using nightly/master or latest release?

I'm using latest release (0.6).

**Comment by hkupty** [comment]
Merged #42 and it should have fixed it. Thanks a lot for the issue and PR folks!

**Comment by hkupty** [comment]
I'm closing this since it seems to have been fixed.
Thanks a lot everyone for helping making nvimux better :)
